src/components/Kv/KvAccordion.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+	<div
+		class="kv-accordion"
+		:class="{
+			'kv-accordion--open' : isOpen,
+		}"
+	>
+		<button
+			class="kv-accordion__header-button"
+			@click.prevent="toggle"
+			:disabled="disabled"
+			:aria-controls="`kv-accordion-${id}`"
+			:aria-expanded="isOpen ? 'true' : 'false'"
+		>
+			<div class="kv-accordion__header-text">
+				<slot name="header"></slot>
+			</div>
+			<kv-icon
+				class="kv-accordion__header-icon"
+				name="small-chevron"
+				:from-sprite="true"
+				aria-hidden="true"
+			/>
+		</button>
+		<kv-expandable>
+			<div
+				class="kv-accordion__pane"
+				:id="`kv-accordion-${id}`"
+				v-show="isOpen"
+				:aria-hidden="isOpen ? 'false' : 'true'"
+			>
+				<slot></slot>
+			</div>
+		</kv-expandable>
+	</div>
+</template>
+
+<script>
+// Accordion a11y resources
+// https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html
+// https://www.aditus.io/patterns/accordion/
+
+// Future improvement
+// Currently the slot content is inside the button, which means h2, h3 etc. won't be
+// navigatable via headings. See https://daverupert.com/2019/12/why-details-is-not-an-accordion/
+// h2 + button // ✅ H1 will show up when navigating by headings
+// button + h2 // ❌ H1 will not show up when navigating by headings
+// Perhaps we could do some magic DOM reordering via this.$slots.header or
+// pass a prop like 'tag' that sets the parent node of the button. <accordion tag="h3">...
+
+import KvExpandable from '@/components/Kv/KvExpandable';
+import KvIcon from '@/components/Kv/KvIcon';
+
+export default {
+	components: {
+		KvExpandable,
+		KvIcon,
+	},
+	props: {
+		/**
+		 * Unique id. used for a11y
+		* */
+		id: {
+			type: String,
+			required: true,
+			validator: v => v.length > 0 && !/\s/g.test(v), // must be a valid html5 id
+		},
+		/**
+		 * Whether the body is shown initially
+		* */
+		open: {
+			type: Boolean,
+			default: false,
+		},
+		/**
+		 * Whether the accordion can be toggled
+		* */
+		disabled: {
+			type: Boolean,
+			default: false,
+		},
+	},
+	data() {
+		return {
+			isOpen: this.open
+		};
+	},
+	methods: {
+		toggle() {
+			if (!this.disabled) {
+				this.isOpen = !this.isOpen;
+				/**
+				 * Fires when the accordion has been toggled.
+				 * Contains an object with a boolean 'open' property of the current open state of the accordion
+				 * @event toggle
+				 * @type {Event}
+				 */
+				this.$emit('toggle', { open: this.isOpen });
+			}
+		},
+		expand() {
+			if (!this.disabled) {
+				this.isOpen = true;
```

Comment:
We need at least a generic tracking event on the clicks for opening and closing these.

Reply:
I was thinking that tracking would be handled by the parent of these components, by hooking into the 'toggle' event.

Based on what we have now, 
```

// FrequentlyAskedQuestions.vue
<kv-accordion @toggle="trackFAQInteraction" ...>
    <template v-slot:title><h3>{{title}}</h3></template>
    <div>lorem ipsum...</div>
</kv-accordion>

trackFAQInteraction(event) {
    if (event.open) {
        this.$kvTrackEvent(this.analyticsCategory, 'click-faq-expand', this.title);
    }
}


// LoanDetailsPanel.vue
<div>
<kv-accordion @toggle="trackLoanInteraction" :id="panelId">
    <template v-slot:title><h3>{{title}}</h3></template>
    <div>lorem ipsum...</div>
</div>

trackLoanInteraction(event) {
    this.$emit('track-interaction', {
        interactionType: `${event.open ? 'open' : 'close'}-${this.panelId}-panel`,    
    }
}
```

Reply:
Ok, I suppose that works for now.